# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../uclibc/mempcpy.patch
# Copyright (C) 2006 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

many __symbols are now hidden in uClibc, and at least __mempcpy is
used by some apps, so keep compatibility
  Michael Tross

--- ./include/string.h.old	2006-09-26 18:32:31.000000000 +0200
+++ ./include/string.h	2006-09-27 21:30:52.000000000 +0200
@@ -235,6 +235,7 @@
 			__const void *__restrict __src, size_t __n)
      __THROW __nonnull ((1, 2));
 #endif
+#define __mempcpy mempcpy
 extern void *mempcpy (void *__restrict __dest,
 		      __const void *__restrict __src, size_t __n)
      __THROW __nonnull ((1, 2));
